src/Tasks.js

```diff
@@ -0,0 +1,61 @@
+class Tasks {
+    static tasks = [];
+
+    constructor(id, description) {
+      // this.tasks=[];
```

Comment:
I think maybe this is unused code?  Perhaps you can remove it and there will not be any problem.